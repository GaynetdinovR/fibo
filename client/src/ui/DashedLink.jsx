import styles from "../styles/Ui.module.sass";
import classNames from "classnames";

const DashedLink = ({ children, className, onClickFn }) => {
	return (
		<button onClick={() => onClickFn()} className={classNames(className, styles.dashed_link)}>
			{children}
		</button>
	);
};

export default DashedLink;