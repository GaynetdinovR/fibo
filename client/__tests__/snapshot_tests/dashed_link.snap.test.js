import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import DashedLink from "../../src/ui/DashedLink.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<DashedLink>DashedLink</DashedLink>).toJSON();
	expect(tree).toMatchSnapshot();
});
