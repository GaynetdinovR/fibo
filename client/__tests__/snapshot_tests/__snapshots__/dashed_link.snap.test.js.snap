// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="dashed_link"
  onClick={[Function]}
>
  DashedLink
</button>
`;
